styleguide/source/_patterns/00-atoms/09-icons/05-search.twig

```diff
@@ -1 +1 @@
-<span class="icon icon-search" data-grunticon-embed></span>
```

Comment:
Can you explain what removing `data-grunticon-embed` is doing here?

Reply:
it was interfering with the default background image

Reply:
right - I'm familiar with how you did this because I saw it while you were working on it. Would you mind updating the PR with a little more detail on how this works so that there's a clearer record in the PR for posterity's sake?

Reply:
This PR has been updated per your suggestions